R/connect.R

```diff
@@ -45,7 +45,9 @@ Connect <- R6::R6Class(
     #' @param server The base URL of your Posit Connect server.
     #' @param api_key Your Posit Connect API key.
     initialize = function(server, api_key) {
-      message(glue::glue("Defining Connect with server: {server}"))
+      if (!is_testing()) {
```

Comment:
Another place we could add a helper function instead of wrapping with the `if` everywhere (though is this it?)

Reply:
There are only two of these `if (!is_testing())` blocks for the two messages that were by far the chattiest. I don't mind adding helper though.